### PR TITLE
ADEN-9330 | Non lazy loaded incontent boxads

### PR DESCRIPTION
**Pull request by bckatarzyna**

### CODE REVIEW

File: app/modules/ads/setup.js [hunk 1]
```diff
@@ -157,6 +157,7 @@ export const adsSetup = {
     context.set('options.video.moatTracking.sampling', instantConfig.get('wgAdDriverPorvataMoatTrackingSampling'));
 
     context.set('options.gamLazyLoading.enabled', instantConfig.isGeoEnabled('wgAdDriverGAMLazyLoadingCountries'));
+    context.set('options.nonLazyIncontents.enabled', instantConfig.get('icNonLazyIncontents'));
```
Comment (dmnsobczak) [inline app/modules/ads/setup.js:163]: Let's not forget to add it to ICBM

File: app/modules/ads/setup.js [hunk 2]
```diff
@@ -327,6 +328,12 @@ export const adsSetup = {
       }
     }
 
+    if (context.get('options.nonLazyIncontents.enabled')) {
+      context.set('options.useTopBoxad', true);
```
Comment (dmnsobczak) [inline app/modules/ads/setup.js:333]: Maybe it's a good decision to do top_boxad on mobile cleanup first or in this ticket?

File: app/modules/ads/setup.js (hunk 1 above)
Comment (bckatarzyna) [reply app/modules/ads/setup.js:163]: Added ✔️

File: app/modules/ads/setup.js (hunk 2 above)
Comment (bckatarzyna) [reply app/modules/ads/setup.js:333]: I will create a separate ticket for this

File: app/modules/ads/slots-loader.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    context.set(
+      'bidders.prebid.bidsRefreshing.bidsBackHandler',
+      () => {
+        const slotId = this.getSlotId();
+
+        utils.logger(logGroup, `refresh ended for: ${slotId}`);
+        context.get(`bidders.prebid.bidsRefreshing.${slotId}.resolve`)();
+        bidsBackHandler();
```
Comment (dmnsobczak) [inline app/modules/ads/slots-loader.js:33]: I wish we could make a queue or an array on `bidders.prebid.bidsRefreshing.bidsBackHandler`, but it's a major :< For experiment purposes: it's fine.

**Review by dmnsobczak — APPROVED** [review]

**Comment by dmnsobczak** [comment]
@bckatarzyna Please fix linter error

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**